src/api/procedures/addInstruction.ts

```diff
@@ -104,10 +105,31 @@ async function getTxArgsAndErrors(
     endBlockErrIndexes: number[];
     datesErrIndexes: number[];
   };
-  /* eslint-disable @typescript-eslint/no-explicit-any */
-  addAndAffirmInstructionParams: any;
-  addInstructionParams: any;
-  /* eslint-enabled @typescript-eslint/no-non-null-assertion */
+  addAndAffirmInstructionParams: [
+    u64,
+    SettlementType,
+    Moment | null,
+    Moment | null,
+    {
+      from: PortfolioId;
+      to: PortfolioId;
+      asset: Ticker;
+      amount: Balance;
+    }[],
+    PortfolioId[]
+  ][];
+  addInstructionParams: [
+    u64,
+    SettlementType,
+    Moment | null,
+    Moment | null,
+    {
+      from: PortfolioId;
+      to: PortfolioId;
+      asset: Ticker;
+      amount: Balance;
+    }[]
+  ][];
```

Comment:
Name these types outside the function to avoid so much repetition (`AddAndAffirmInstructionParams` and `AddInstructionParams`)